docs/content/guides/getting-started/installation.md

```diff
@@ -26,7 +47,6 @@ To install Handsontable using a framework, see:
  - [Installation in Vue 2](@/guides/integrate-with-vue/vue-installation.md)
  - [Installation in Vue 3](@/guides/integrate-with-vue3/vue3-installation.md)
 :::
-:::
```

Comment:
I do not notice anything out of order, everything is visible as intended here.

Reply:
Then this page needs to be improved, because:

- information how to set up the stylesheet is missing from the React Installation page
- the "`HotTable` component" heading is present in Markdown but not rendered in the React Installation page

Reply:
Header is indeed missing. I will add snippet with css import.